src/components/loaders/ShowPosterLoader.tsx

```diff
@@ -0,0 +1,27 @@
+import Skeleton from 'react-loading-skeleton';
+import 'react-loading-skeleton/dist/skeleton.css';
+
+interface ShowCardLoaderProps {
+    /**
+     * Number of skeleton loaders to display
+     */
+    count: number;
+}
+
+/**
+ * A skeleton loader of the show poster component. To be rendered while
+ * main component is loading.
+ * @param count | number of poster placeholders to be rendered
+ * @returns {JSX.Element}
+ */
+export default function ShowPosterLoader({ count }: ShowCardLoaderProps): JSX.Element {
```

Comment:
With this interface, would it be better to define the type in the component? `ShowPosterLoader({ count }: number)`

Reply:
I am not sure that either would be ojectively better in this case. Since this is a component I am more sticking to the pattern we have been using of creating an interface. While it probably does take up more room than is necessary, it keeps the components more consistent. I think we could do `ShowPosterLoader({ count }: {count: number })` though and it would be functionally identical.